live-examples/css-examples/backgrounds-and-borders/border-radius.css

```diff
@@ -0,0 +1,7 @@
+#example-element {
+    width: 200px;
```

Comment:
I think we should use something like `width: 80%;` here - as it is this cuts off the end of the text.